GNUmakefile

```diff
@@ -59,3 +58,5 @@ clang $(COVERAGE_TARGETS) $(COVERITY_TARGETS) cppcheck indent:
 		COVLEVEL=$(COVLEVEL)			\
 		CPPCHECK_ARGS=$(CPPCHECK_ARGS)		\
 		-C devel "$@"
+
+FORCE:
```

Comment:
Should FORCE be phony?

Reply:
I don't think it needs to be, but the docs don't really say.  What they do say is that the prerequisites of PHONY can't be patterns because '%' is always interpreted literally, and that because the FORCE target has no prerequisites or rules, `make` will assume it has always been updated which will force anything depending on it to be rebuilt.